content/guides/author-libraries.md

```diff
@@ -34,56 +77,70 @@ export function wordToNum(word) {
 };
 ```
 
-The usage spec for the library will be as follows.
+The usage specification for the library use will be as follows:
 
 ```javascript
-import * as webpackNumbers from 'webpack-numbers';
-
+import * as webpackNumbers from 'webpack-numbers';//ES2015 module import
```

Comment:
Nitpicking, but can you add aspace *before* the comments?
Also decide on whether or not to use a space after the `//`, as currently both seem to be used ;)

Reply:
In the other guides the comments are used in the following way:
`// Comment text`
 `var someJSCode={}`
Do you mind if I set the comments format in that way?

Reply:
> Also decide on whether or not to use a space after the `//`, as currently both seem to be used ;)

@TheDutchCoder yeah we should probably try to standardize it by padding both sides.

> Do you mind if I set the comments format in that way?

That sounds good to me, though I'd probably pad that `=` sign as well (e.g. `var someJSCode = {}`) 😜.